### PR TITLE
Update basic-types.markdown

### PR DESCRIPTION
**Pull request by SherSpock**
Small changes for accuracy and flow

### CODE REVIEW

File: getting-started/basic-types.markdown [hunk 1]
```diff
@@ -381,8 +381,8 @@ iex> elem(tuple, 1)
 "hello"
 ```
 
-When "counting" the number of elements in a data structure, Elixir also abides by a simple rule: the function is named `size` if the operation is in constant time (i.e. the value is pre-calculated) or `length` if the operation is linear (i.e. calculating the length gets slower as the input grows). As a mnemonic, both "length" and "linear" start with "l".
+When quantifying the number of elements in a data structure, Elixir also abides by a simple rule: the function is named `size` if the operation is in constant time (i.e. the value is pre-calculated) or `length` if the operation is linear (i.e. calculating the length gets slower as the input grows). As a mnemonic, both "length" and "linear" start with "l".
```
Comment (josevalim) [inline getting-started/basic-types.markdown]: As a non-native speaker, "counting" sounds much clearer than "quantifying". Do we have a simpler alternative here? :)

Comment (eksperimental) [inline getting-started/basic-types.markdown]: maybe just "when counting the elements"

Comment (SherSpock) [inline getting-started/basic-types.markdown]: @josevalim @eksperimental I agree! I was trying to preserve what I thought was the the intent of the writer, which seemed to me to be to draw attention to the fact that Elixir is not counting the way a human would. But ease of comprehension is definitely better :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**